src/poker.js

```diff
@@ -1,5 +1,23 @@
 class Poker {
-  // Write your implementation here
+  winningPair(player1, player2) {
```

Comment:
You will need the object to convert J, Q, K, A into numbers, or else you cannot compare which letters are stronger